nexus/json/createRepos.json

```diff
@@ -16,6 +16,7 @@
   repository.createNpmProxy('npm-registry', 'https://registry.npmjs.org', 'default', true);
   repository.createNpmHosted('npm-private', 'npm-private', true, WritePolicy.ALLOW_ONCE);
   repository.createNpmGroup('npm-all', ['npm-registry', 'npm-private'], 'default');
```

Comment:
Did you check if `nom-all` is still in use somewhere? If not, I think we should remove it. Maybe you can open a separate issue/PR with this change?

Reply:
We did not see it in use. Will verify and if it is unused, create an issue/PR for that.